app/src/androidTest/java/de/test/antennapod/service/playback/PlaybackServiceMediaPlayerTest.java

```diff
@@ -129,7 +129,7 @@ public void testInit() {
 
     private Playable writeTestPlayable(String downloadUrl, String fileUrl) {
         Feed f = new Feed(0, null, "f", "l", "d", null, null, null, null, "i", null, null, "l", false);
-        FeedPreferences prefs = new FeedPreferences(f.getId(), false, FeedPreferences.AutoDeleteAction.NO, VolumeAdaptionSetting.OFF, null, null);
+        FeedPreferences prefs = new FeedPreferences(f.getId(), false, FeedPreferences.AutoDeleteAction.NO, VolumeAdaptionSetting.OFF, FeedPreferences.SkipInboxSetting.NO, null, null);
```

Comment:
I would rename the setting to something like `NewEpisodesAction.NOTHING` and `NewEpisodesAction.ADD_TO_INBOX`. Then we can also add something like `NewEpisodesAction.ENQUEUE` (see #5246).

Reply:
I have finished renaming this option, and pushed my changes.

Does this option belong in the "User Interface" menu, or should it be moved to "Network"?